security/medusa/l2/acctype_path_access.c

```diff
@@ -144,62 +150,67 @@ char *medusa_get_path(const struct path *path, const struct qstr *last, int last
 			diff = strlen(slash);
 			*slash = '\0';
 			while (slash >= pathbuf) {
-				*(slash + diff) = *slash;
```

Comment:
*(slash + diff) = *slash sounds me better: 1) left side has the same form as the right one 2) pointer arithmetic is normal using index zero... why is not enough clear using another index, too?
slash[diff] = * slash: left side uses [], right side uses pointer arithmetic...

Reply:
would `slash[diff] = slash[0]` be better?